packages/hardhat-core/test/builtin-tasks/flatten.ts

```diff
@@ -327,6 +327,7 @@ contract AWithLicense {}`
             license: "MIT",
           }
         );
+        console.log(allFlattened)
```

Comment:
I guess this was temporary?

Reply:
Was removed on https://github.com/nomiclabs/hardhat/pull/1223/commits/2a291330e6c5d070241256c5059899fc7bc5b00f